setup.py

```diff
@@ -68,7 +68,7 @@ def get_package_data(package):
       include_package_data=True,
       packages=get_packages('defender'),
       package_data=get_package_data('defender'),
-      install_requires=['Django>=1.8,<2.2', 'redis>=2.10.3,<=3.2'],
+      install_requires=['Django>=1.8,<2.2', 'redis>=2.10.3,<=3.2.1'],
```

Comment:
LGTM, I wonder if we want to change to 3.2.* so that we can get all patches automatically, vs having to hard code this version every time. I can see pros and cons for both approaches, wondering what people thing.

Reply:
I just merged another PR that conflicts with this one on this same line, can you rebase your PR so that it merges cleanly, please.

Reply:
Rebase done. Also change Redis version to be `<3.3` if ok?

Reply:
also once this PR is merged let me know when you will publish a new version to I can upgrade my project to it if ok? Ty

Note: I already upped the version number in setup.py for you to save time